src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -263,6 +263,24 @@
      * or enum declaration (interfaces cannot have static initializers).  The
      * first and only child is a statement list.
      *
+     * <p>For Example:</p>
+     * <pre>
+     * static {
+     *   num = 10;
+     * }
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * --STATIC_INIT -&gt; STATIC_INIT
```

Comment:
please remove leading `--` they are not required

Reply:
Done.